src/tests/testSBuf.cc

```diff
@@ -106,9 +107,21 @@ class TestSBuf : public CPPUNIT_NS::TestFixture
     void testStdStringOps();
     void testIterators();
     void testSBufHash();
+    void testAsHex();
 };
 CPPUNIT_TEST_SUITE_REGISTRATION( TestSBuf );
 
+class TestIoManip : public CPPUNIT_NS::TestFixture
```

Comment:
IoManip is unrelated to SBuf. We should not add this new test into testSBuf.cc, even if the test is using SBuf.

Reply:
>  this can be some throwaway tests for development and this is a handy place to place them.

You would need to define the implications of "throwaway tests for development" for this argument to be usable in a discussion. For example, if the implication is that these tests should be thrown away at the end of this PR, then please say so. Removing them will obviously address this change request.

>  It could also be something we could be interested into as a model moving forward, to get out of the stub hell we currently have to maintain

If you want to propose a "model" change, please do so first/separately/properly. I am happy to discuss such a proposal. Until then, arguments based on a new but insufficiently specified model are not admissible IMO.

Reply:
I agree with Alex on this one. Please move this test to a separate unit -see `src/Makefile.am` line 807 and later for where to place the tests of `src/base/` code.

Reply:
@rousskov went ahead and implemented it while I was away. Thanks for doing that!

Reply:
Sigh.. I crossed wires with html_quote. Too much enthusiasm